packages/data/config/datatypes_conf.xml.sample

```diff
@@ -0,0 +1 @@
+../../../lib/galaxy/config/sample/datatypes_conf.xml.sample
```

Comment:
You're going to need to put this in either the MANIFEST or package data for setup.py also I think.

Reply:
where do you think it makes more sense?

Reply:
setup.py I think

Reply:
So from I've read it seems to be needed in both places. Please correct me if I'm wrong.

Reply:
> So from I've read it seems to be needed in both places. Please correct me if I'm wrong.

Both is the safest, see https://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files

Reply:
I think this file should be `packages/data/galaxy/config/sample/datatypes_conf.xml.sample`